project/MiMa.scala

```diff
@@ -1152,10 +1152,27 @@ object MiMa extends AutoPlugin {
       //  * this list ends with the latest released version number
       //  * is kept in sync between release-2.4 and master branch
     )
+    
+    val Release25Filters = Seq(
+      "2.5.0" -> Seq(
+          
+          // #22759 LMDB files
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.valueBuffer_="),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.db"),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.keyBuffer"),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.env"),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.ddata.LmdbDurableStore.valueBuffer"),
+          
+          ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.traversalBuilder"),
+          ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.named"),
+          ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.addAttributes"),
+          ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.async")
```

Comment:
I have no clue of why these stream filters are needed now. Why were they not needed when we released 2.5.0? Have we changed anything there?

```
[info] akka-stream: found 4 potential binary incompatibilities while checking against com.typesafe.akka:akka-stream_2.11:2.4.12  (filtered 189)
[error]  * abstract method traversalBuilder()akka.stream.impl.TraversalBuilder in trait akka.stream.Graph is inherited by class GraphModule in current version.
[error]    filter with: ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.traversalBuilder")
[error]  * abstract method named(java.lang.String)akka.stream.Graph in trait akka.stream.Graph is inherited by class GraphModule in current version.
[error]    filter with: ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.named")
[error]  * abstract method addAttributes(akka.stream.Attributes)akka.stream.Graph in trait akka.stream.Graph is inherited by class GraphModule in current version.
[error]    filter with: ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.addAttributes")
[error]  * abstract method async()akka.stream.Graph in trait akka.stream.Graph is inherited by class GraphModule in current version.
[error]    filter with: ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("akka.stream.Graph.async")
```